test/suites/dev/moonbase/test-fees/test-length-fees2.ts

```diff
@@ -2,6 +2,39 @@ import "@moonbeam-network/api-augment";
 import { describeSuite, expect } from "@moonwall/cli";
 import { createViemTransaction } from "@moonwall/util";
 import { ConstantStore } from "../../../../helpers/constants";
+import { hexToU8a } from "@polkadot/util";
+
+// EIP-7623 constants
+const TOTAL_COST_FLOOR_PER_TOKEN = 10n;
+const COST_FLOOR_PER_ZERO_BYTE = TOTAL_COST_FLOOR_PER_TOKEN;
+const COST_FLOOR_PER_NON_ZERO_BYTE = 4n * TOTAL_COST_FLOOR_PER_TOKEN;
+const STANDARD_COST_PER_ZERO_BYTE = 4n;
+const STANDARD_COST_PER_NON_ZERO_BYTE = 16n;
+const BASE_TX_COST = 21000n;
```

Comment:
Same, lets move these to the constants file, may be usable by other tests in the future.

Reply:
Addressed in https://github.com/moonbeam-foundation/moonbeam/pull/3441/commits/b69b9d2de2a6caa2145242388ac181a721c7f971